.github/workflows/CI-tag-orchestrator.yml

```diff
@@ -3,7 +3,7 @@ name: Tag-orchestrator
 on:
   push:
     tags:
-      - '[0-9].[0-9].[0-9]*'
+      - '[0-9]+.[0-9]+.[0-9]+-[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
This regex enforces the following format for all the tags 
`DIGIT.DIGIT.DIGIT-DIGIT.DIGIT-DIGIT` that means nothing like D.D.D will match anymore. 

If you want to support both cases try to use the following regex and see if GH actions will accept the syntax. `\` in the regex to escape `.` since it is an actual DOT vs any character from regex syntax 

```^[0-9]+\.[0-9]+\.[0-9]+(-[0-9]+\.[0-9]+\.[0-9]+)?$```

Reply:
Yes that's correct, on the GitHub tags side there will be nomore `X.Y.Z` but only `X.Y.Z-A.B.C`; slighlty different for Docker tags.